src/ui/PickColor.tsx

```diff
@@ -0,0 +1,24 @@
+import { useEffect, useState } from 'react'
+import { HexColorPicker } from 'react-colorful'
+import { setState, useStore } from '../store'
+
+export function PickColor(): JSX.Element {
+  const [color, setColor] = useState('#FFFF00')
```

Comment:
We should use the color from the store as the initial value.

Reply:
Updated to pick the initial yellow from the store.
Let me know if it's correct.